src/components/04-table/table--stacked.njk

```diff
@@ -0,0 +1,138 @@
+<div class="width-mobile">
+
+  <table class="usa-table usa-table--stacked">
+    <caption>Stacked bordered table</caption>
+    <thead>
+      <tr>
+        <th scope="col">Document title</th>
+        <th scope="col">Description</th>
+        <th scope="col">Year</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th data-label="Document title" scope="row">Declaration of Independence</th>
```

Comment:
Any possibility that `data-label` could be too generic and possibly run into collisions with scripts in projects? Seems like an edge case wrapped in an edge case but thought I'd flag because I really don't know.

Reply:
For this, I followed the CFPB's pattern, which uses `data-label`: https://cfpb.github.io/design-system/components/tables#responsive-stacked-table-with-header-directory-tables

You're right, this `data-label` attribute is also used in the date-picker component, but I don't see a situation where they would interact. The CSS here is looking for a `td` or `th` with the `data-label` attribute.

Reply:
I'm comfortable with this attribute and scope